Zc-specification/Zc.adoc

```diff
@@ -120,6 +121,22 @@ Therefore common ISA strings can be updated as follows to include the relevant Z
 * RV32IMC becomes RV32IM_Zce
 * RV32IMCF becomes RV32IMF_Zce
 
+[#misaC]
+=== MISA.C
+
+If all the instructions from the C extension are present, and the MISA CSR, has been implemented, then MISA.C is set. This happens if the following extensions are selected:
+
+* Zca and not F
+* Zca, Zcf and F is specified (RV32 only)
+* Zca, Zcf and Zcd if D is specified (RV32 only)
+** this configuration excludes Zcmp, Zcmt
+* Zca, Zcd if D is specified (RV64 only)
+** this configuration excludes Zcmp, Zcmt
+
+Therefore MISA.C can be set without directly specifying the C extension.

```

Comment:
I think this sentence is going to confuse people.  If misa.C is set, then C _is_ present.  "Directly specified" or not isn't an ISA concept and doesn't really matter.